docker/www/Dockerfile

```diff
@@ -34,13 +47,16 @@ RUN apt-get update && \
   apt-get autoremove -y
 
 COPY data-vol /etc/logrotate.d
-COPY --chown=root:www-data www $APACHE_ROOT
+COPY --from=builder --chown=root:www-data /var/lib/jenkins/www $APACHE_ROOT
+COPY --from=builder /var/lib/jenkins/www/profiles/bibdk/modules/bibdk_config/docker/settings.php $APACHE_ROOT/sites/default
+RUN chown root:www-data $APACHE_ROOT
+#COPY --chown=root:www-data www $APACHE_ROOT
 COPY conf/000-default.conf /etc/apache2/sites-enabled
 COPY script/entry.sh /
 COPY script/healthcheck.sh /
 COPY script/apache_ready.sh /
 COPY script/run_environment.sh /
-COPY www/profiles/bibdk/modules/bibdk_config/docker/settings.php $APACHE_ROOT/sites/default
+ADD --chown=www-data:www-data ["https://is.dbc.dk/job/Bibliotek%20DK/job/Tools/job/Fetch%20Bibliotek%20DK%20files%20folder/lastSuccessfulBuild/artifact/files.tar.gz","/tmp"]
```

Comment:
Kan du et eller andet sted her i Dockerfilen lave en WORKDIR $APACHE_ROOT.

Så når vi exec'er ind i containeren, så står vi allerede i /var/www/html og skal ikke til først og cd'e os derind. Det er et irritationsmoment for mig, hver gang at skulle gøre det.